### PR TITLE
Fleshing out updated roadmap

### PR DESCRIPTION
**Pull request by bladey**
Updating current roadmap to the new format, with @cursedimg's help.

### CODE REVIEW

File: docs-next/pages/roadmap.mdx [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
+Over the last six months, we’ve built around the existing stable core of Keystone 5's packages to test the changes we wanted to make and validate our ideas. Now, in Keystone Next, we've substantially rewritten the CLI, Schema config, and Admin UI to make them more powerful and easier to use than ever before.
 
-In the last six months, we built around the existing stable core of Keystone 5's packages to test
-the changes we wanted to make and validate our ideas. We're now happy with what we've built, and
-over the course of this year, Keystone Next will become Keystone 6.
+There is a lot more to do, and with a larger than ever team at [Thinkmill](https://www.thinkmill.com.au) working on it, we're excited to show you what we're planning and get it shipped!
 
-There is a lot to do, and with a larger than ever team at [Thinkmill](https://www.thinkmill.com.au)
-working on it, we're excited to show you what we're planning and get it shipped!
+We'll be updating this roadmap fortnightly (or as often as we need to) to track progress and plans, find answers or info that matters to your project, and communicate with us more dynamically while we grow the platform.
```
Comment (timleslie) [inline docs-next/pages/roadmap.mdx:13]: This sentence gives my brain a parse error. We start in first person with talk about `We'll be...`, then transition into actions like `find answers` and `communicate with us` that suggest we're now in second person.

File: docs-next/pages/roadmap.mdx [hunk 2]
```diff
@@ -78,12 +86,16 @@ streamlined migration stories for.
 
 ## Field Types Review
 
+<Status look="notStarted">Not started</Status>
```
Comment (timleslie) [inline docs-next/pages/roadmap.mdx:89]:
```suggestion
<Status look="notStarted">Figuring it out</Status>
```

File: docs-next/pages/roadmap.mdx [hunk 3]
```diff
@@ -78,12 +86,16 @@ streamlined migration stories for.
...
+
 We're planning to revisit our field types and make them more powerful and consistent. This will
 include new interface options for the Admin UI, new features for validation and logic, and better
 `null` value handling.
 
 ## Nested (JSON) Fields
 
+<Status look="figuringItOut">Figuring it out</Status>
```
Comment (timleslie) [inline docs-next/pages/roadmap.mdx:97]:
```suggestion
<Status look="figuringItOut">Not started</Status>
```

File: docs-next/pages/roadmap.mdx [hunk 4]
```diff
@@ -93,19 +105,25 @@ stored in a JSON field in the database.
 
 ## Singletons
 
+<Status look="theresAPlan">There's a plan</Status>
```
Comment (timleslie) [inline docs-next/pages/roadmap.mdx:108]:
```suggestion
<Status look="theresAPlan">Not started</Status>
```

**Review by timleslie — CHANGES_REQUESTED** [review]
Looking pretty good so far!

File: docs-next/pages/roadmap.mdx (hunk 1 above)
Comment (cursedimg) [reply docs-next/pages/roadmap.mdx:13]: Totally agree, corrections incoming

**Comment by cursedimg** [comment]
Hey @JedWatson - updates in and ready for your final review! 🙏

**Review by JedWatson — APPROVED** [review]
Looks great! 🙌

**Review by timleslie — APPROVED** [review]

**Pull request merged**